VKApp/VKApp/Models/Posts/Post.swift

```diff
@@ -0,0 +1,25 @@
+// Post.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Posts
+final class Posts: Decodable {
+    var items: [PostItems]?
+    var group: [Groups]?
+    var profile: [Item]?
```

Comment:
```suggestion
    var profiles: [UserItem]?
```
Можно сделать чтобы все типы имели похожий нейминг
А то пост это постайтем
Группы это группы
А пользователи просто айтем 
Либо назвать пост группа юзер и т.д
Либо постайтем и т.д
Так будет читабельней, я думаю.